BIPs/2023-W39/BIP-442-permission-preset2.json

```diff
@@ -0,0 +1,262 @@
+{
+  "version": "1.0",
+  "chainId": "1",
```

Comment:
Note that the safe address is missing here and it is no where specified in the BIP to which safe this payload should be loaded.

Could you please update the payload as well as including this information in the form post somewhere?

Reply:
Done